hugo/content/050-tutorials/20-developer/_index.md

```diff
@@ -0,0 +1,34 @@
+---
+title: App Developer
+url: /using-gardener/developer/topic
```

Comment:
can we add the next level of titles to the TOC for this chapter ?

Reply:
Personally, I wouldn't do that. Here the grouping by "theme" or "experience", which is provided by the overview page, would then be missing. Sorting the TOC is also not quite easy...it would then only be a "stupid listing" without a content structure.